### PR TITLE
[WIP] Refactor Webindex for move to Apache.

### PR DESCRIPTION
**Pull request by cjmctague**
This is a work in progress. 
I've started with just Webindex and would like to get some feedback before starting the other two examples.

### CODE REVIEW

File: webindex/modules/core/pom.xml [hunk 1]
```diff
@@ -1,29 +1,28 @@
...
+  the License.
 -->
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
   <modelVersion>4.0.0</modelVersion>
   <parent>
-    <groupId>io.github.astralway</groupId>
-    <artifactId>webindex-parent</artifactId>
-    <version>0.0.1-SNAPSHOT</version>
+    <groupId>org.apache.fluo</groupId>
+    <artifactId>fluo-webindex-parent</artifactId>
```
Comment (ctubbsii) [inline webindex/modules/core/pom.xml]: We don't need to change the artifactIds if we don't want to.

File: webindex/modules/core/src/main/java/org/apache/fluo/webindex/core/Constants.java [hunk 2]
```diff
@@ -1,18 +1,19 @@
 /*
- * Copyright 2015 Webindex authors (see AUTHORS)
- * 
- * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
- * in compliance with the License. You may obtain a copy of the License at
- * 
+ * Licensed to the Apache Software Foundation (ASF) under one or more contributor license
```
Comment (ctubbsii) [inline webindex/modules/core/src/main/java/org/apache/fluo/webindex/core/Constants.java:2]: Some of these license headers seem to have inconsistent wrapping. I wonder if we can fix that with smart use of license-maven-plugin in future versions of the fluo-parent POM.

**Review by ctubbsii — COMMENTED** [review]
I don't think it's necessary to prefix the artifactIds with "fluo-". It's a bit excessive. The term "fluo" is already in the groupIds.

File: webindex/modules/core/pom.xml (hunk 1 above)
Comment (cjmctague) [reply webindex/modules/core/pom.xml]: I do agree the "fluo-" prefix is excessive. I added it just to keep it consistent across all the Fluo projects.

Comment (ctubbsii) [reply webindex/modules/core/pom.xml]: Understood, but it's not necessary. We already break that pattern with the 'build-resources' artifact anyway.

**Comment by ctubbsii** [comment]
This was presented as a WIP, but I think a lot of it has been superseded by #6 (I didn't realize this was still open when I created that one). I think I may have missed one or two things, though.

**Pull request closed without merging**